models/location_model.go

```diff
@@ -73,9 +73,10 @@ func (s *BackupLocation) AfterFind() error {
 
 // S3LocationConfig contains required properties for accessing S3 Bucket
```

Comment:
@Dasio JFY I decided to rename files to preserve naming consistency (as per discussion in Slack).